#1 Homework

```diff
@@ -0,0 +1,27 @@
+var array = [3, 5, 8, 2, 1, 0, 0, 4]
+
+var a = 0;
+var b = array[0];
+var c = 0;
+var maxd = 0;
+var d = array[0];
+
+for (var i = 0; i < array.length; i++) {
+    a = array[i] + a; // a = hodnota v poli + a. Po prvni iteraci; a = 3 + 0 -> a = 3. Po druhe iteraci; a = 5 + 3 -> a = 8.
```

Comment:
If you use a certain value, that you need to access via a getter, it is better to use it as a variable

Reply:
so here it would be better to have a variable called ie. current = array[i] and then replace all of the array[i] with current

Reply:
Updated:
- var j = array[i];